packages/wdio-junit-reporter/src/index.js

```diff
@@ -14,6 +14,7 @@ class JunitReporter extends WDIOReporter {
         this.suiteNameRegEx = this.options.suiteNameFormat instanceof RegExp
             ? this.options.suiteNameFormat
             : /[^a-zA-Z0-9]+/
+        this.isCucumber = this.options.isCucumber
```

Comment:
Where does this come from?

Reply:
Would be set in `wdio.conf.js`...

```
reporters.push(['junit', {
    outputDir: junitOutputDir,
    isCucumber: true,
    errorOptions: {
        error: 'message',
        failure: 'message',
        stacktrace: 'stack'
    }
}]);
```

Reply:
@christian-bromann I also have to reinstate the original code in a conditional before this is good to go. But you think it should just be an option of this package and not its own package correct? Like we shouldn't have `@wdio/cucumber-junit-reporter`

Reply:
I wonder if we could propagate this information to the reporter automatically because I don't think someone would have the Mocha behavior when running Cucumber.

Reply:
As for the configuration option, I decided to go for opt-in rather than detecting it by checking active framework that way it didn't disrupt those that prefer the original styling.

Reply:
> Like we shouldn't have @wdio/cucumber-junit-reporter

No we shouldn't

Reply:
We can make it automatic propagation if you feel it is a better idea.

Reply:
yeah, otherwise we would need to document this as an option that can be forgotten.

Reply:
yeah, otherwise we would need to document this as an option that can be forgotten.

Reply:
OK, I will take a look, also I am noticing there are a few bugs in my code as well.